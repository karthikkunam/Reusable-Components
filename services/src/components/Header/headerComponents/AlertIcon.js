import React from 'react';

export default class AlertIcon extends React.PureComponent {
  render() {
    return (
      <a className="nav-item-icon alert-icon-container slide-out" id="sidebarCollapse" href="#">
        <span className="alert-icon">
          <span className="alert-icon-circle">
            <span className="alert-icon-count">3</span>
          </span>
        </span>
        <span className="icon icon-bell" />
      </a>
    );
  }
}
