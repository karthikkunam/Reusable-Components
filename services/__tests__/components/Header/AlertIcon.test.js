import React from 'react';
import AlertIcon from '../../../src/components/Header/headerComponents/AlertIcon';
import { shallow } from 'enzyme';

describe('AlertIcon Component', () => {
  it('renders correctly', () => {
    const component = shallow(<AlertIcon/>);
    expect(component).toMatchSnapshot();
  });
});