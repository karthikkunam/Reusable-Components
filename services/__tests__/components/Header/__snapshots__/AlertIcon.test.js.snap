// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertIcon Component renders correctly 1`] = `
<a
  className="nav-item-icon alert-icon-container slide-out"
  href="#"
  id="sidebarCollapse"
>
  <span
    className="alert-icon"
  >
    <span
      className="alert-icon-circle"
    >
      <span
        className="alert-icon-count"
      >
        3
      </span>
    </span>
  </span>
  <span
    className="icon icon-bell"
  />
</a>
`;
